elmslie-samples/compose-paging/src/main/java/vivid/money/elmslie/samples/android/compose/view/PagingFragment.kt

```diff
@@ -29,16 +29,16 @@ class PagingFragment : ElmComponentFragment<PagingEvent, PagingEffect, PagingSta
     @Preview
     @Composable
     private fun Screen() {
-        val state by state()
-        val effect by effect()
-        PagingScreen(
-            state = state,
-            effect = effect,
-            onRefresh = { store.accept(Ui.PullToRefresh) },
-            onReloadPage = { store.accept(Ui.ClickReloadPage) },
-            onReloadScreen = { store.accept(Ui.ClickReloadScreen) },
-            onCloseToEnd = { store.accept(Ui.CloseToEnd) }
-        )
+        store.states()
+        store.effects()
+//        PagingScreen(
```

Comment:
Let's create a some kind of list with this and the previous one [TODO](https://github.com/vivid-money/elmslie/pull/163/files/61aed549598314acfe8d9612182c95619adb2da4#diff-99cf14ec25a5e6e6edccada42092550c1cb1242de2ebbf0a0d6c145ed454004e). 
Just to not forget about them